webapp/channels/src/components/post_view/message_attachments/message_attachment/message_attachment.tsx

```diff
@@ -94,26 +105,34 @@ export default class MessageAttachment extends React.PureComponent<Props, State>
         this.mounted = false;
     }
 
-    handleHeightReceivedForThumbUrl = ({height}: {height: number}) => {
+    handleHeightReceivedForThumbUrl = ({height}: { height?: number }) => {
         const {attachment} = this.props;
-        if (!this.props.imagesMetadata || (this.props.imagesMetadata && !this.props.imagesMetadata[attachment.thumb_url])) {
+        if (
+            !this.props.imagesMetadata ||
+            (this.props.imagesMetadata &&
+                !this.props.imagesMetadata[attachment.thumb_url])
+        ) {
             this.handleHeightReceived(height);
         }
     };
 
-    handleHeightReceivedForImageUrl = ({height}: {height: number}) => {
+    handleHeightReceivedForImageUrl = ({height}: { height?: number }) => {
         const {attachment} = this.props;
-        if (!this.props.imagesMetadata || (this.props.imagesMetadata && !this.props.imagesMetadata[attachment.image_url])) {
+        if (
+            !this.props.imagesMetadata ||
+            (this.props.imagesMetadata &&
+                !this.props.imagesMetadata[attachment.image_url])
+        ) {
             this.handleHeightReceived(height);
         }
     };
 
-    handleHeightReceived = (height: number) => {
+    handleHeightReceived = (height?: number) => {
         if (!this.mounted) {
             return;
         }
 
-        if (height > 0) {
+        if (height && height > 0) {
```

Comment:
This can be simplified:
```suggestion
        if (height) {
```